### PR TITLE
added w3school html tutorial in english

### PR DESCRIPTION
**Pull request by aulth**
I have added W3school's HTML tutorial in the english language. This course is very helpful for those who just want to start the web development.

### CODE REVIEW

**Review by shrn01 — CHANGES_REQUESTED** [review]
Please sort the resources alphabetically. Read [contribution guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md) for more information

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -650,6 +650,7 @@
 * [Tailwind CSS Tutorial](https://youtube.com/playlist?list=PL4cUxeGkcC9gpXORlEHjc5bgnIi5HEGhw) - The Net Ninja
 * [Web Development By Doing: HTML / CSS From Scratch](https://www.udemy.com/course/web-development-learn-by-doing-html5-css3-from-scratch-introductory/) - Udemy
 * [What the Flexbox](https://flexbox.io) - Wesbos
+* [HTML for Beginner](https://www.w3schools.com/html/) - W3School
```
Comment (shrn01) [inline courses/free-courses-en.md:653]: Move this resource to line 641 to resolve the lint error

**Review by shrn01 — CHANGES_REQUESTED** [review]

File: courses/free-courses-en.md (hunk 1 above)
Comment (aulth) [reply courses/free-courses-en.md]: I have moved the resource.

**Review by shrn01 — APPROVED** [review]

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -638,6 +638,7 @@
 * [HTML and CSS Crash Course](https://scrimba.com/learn/htmlcss) - Kevin Powell (scrimba)
 * [HTML and CSS Crash For Beginner](https://youtube.com/playlist?list=PL4-IK0AVhVjM0xE0K2uZRvsM7LkIhsPT-) - Kevin Powell (YouTube)
 * [HTML and CSS Tutorials](https://www.youtube.com/playlist?list=PL0eyrZgxdwhwNC5ppZo_dYGVjerQY3xYU) - Dani Krossing
+* [HTML for Beginner](https://www.w3schools.com/html) - W3School
```
Comment (shrn01) [inline courses/free-courses-en.md:641]: This is an interactive tutorial. Please move it to the intereactive tutorials file

**Comment by eshellman** [comment]
duplicate

**Pull request closed without merging**